meson.build

```diff
@@ -205,6 +205,14 @@ endif
 config_h.set('HAVE_SNAP', enable_snap,
              description: 'Define to 1 if Snap support is enabled')
 
+# malcontent support
+enable_malcontent = get_option('malcontent')
+if enable_malcontent
+  malcontent_dep = dependency('malcontent-0', version: '>= 0.7.0')
```

Comment:
Nitpick: The version bump from malcontent 0.3.0 to 0.7.0 shouldn’t happen in `build: Allow malcontent support to be enabled or disabled`, since that commit doesn’t actually introduce a dependency on 0.7.0

Reply:
Ok, lets address version bump in #213 instead.